_gcode/M593.md

```diff
@@ -80,7 +80,7 @@ Use a slicer that provides custom G-code macros for layer change. For example yo
 
 - In *Prusa Slicer* you'll enable **Spiral vase**. Then add the following to your **Printer Settings > After layer change G-code** to run a test range of 15Hz to 60Hz:
   ```
-  M593 F{(layer_num < 2 ? 0 : 15 + 45.0 * (layer_num - 2) / 297)} ; Hz Input Shaping Test
+  M593 F{(layer_z < 2 ? 0 : 15 + 45.0 * (layer_z - 2) / 297)} ; Hz Input Shaping Test
```

Comment:
According to [Prusa's List of placeholders page](https://help.prusa3d.com/article/list-of-placeholders_205643), both `layer_num` & `layer_z` are valid.